templates/advantage/payment-methods/index.html

```diff
@@ -32,6 +32,26 @@ <h1>Ubuntu Advantage for&nbsp;Infrastructure</h1>
     <div class="row">
       <div class="col-12">
         <h2>Payment methods</h2>
+
+        <div id="payment-success" class="p-notification--positive u-hide">
+          <p class="p-notification__response">
+            <span class="p-notification__message"></span>
+          </p>
+        </div>
+
+        <div id="payment-errors" class="p-notification--negative u-hide">
+          <p class="p-notification__response">
+            <span class="p-notification__message"></span>
+          </p>
+        </div>
+
+        <div id="payment-warnings" class="p-notification--caution u-hide">
+          <p class="p-notification__response">
+            <span class="p-notification__message">
+              You have one or more failed payments. Use your saved card to <a class="p-notification__action" id="try-again-button">retry payment</a>. <i id="try-again-spinner" class="p-icon--spinner u-animation--spin is-dark u-hide"></i></span>
+          </p>
+        </div>
+
```

Comment:
Here you could have only one notification and change its class (`--positive`, `--negative`, `--caution`) in the JS. 
(I'm not asking you to change it, it's almost just a matter of taste but just letting you know, I don't know if you've considered)

Reply:
I could make the `--positive` and the `--negative` the same div.

.. but not sure how I would merge the `--negative` and the `--caution` to be the same div as they can show up both at the same time.

But is the merge worth it? I feel like having 3 separate elements is sooo much easier, as I just need to reveal them. Once you only have one element you have to perform more complex operations to update the message and colour of the same div.

Reply:
Oh, I didn't realise the caution and negative could show at the same time. I don't think it's worth it, to change it if you prefer it this way it's was just a suggestion!